src/satosa/micro_services/ldap_attribute_store.py

```diff
@@ -19,88 +20,139 @@ class LdapAttributeStore(satosa.micro_services.base.ResponseMicroService):
     to lookup a person record in LDAP and obtain attributes
     to assert about the user to the frontend receiving service.
     """
+    logprefix = "LDAP_ATTRIBUTE_STORE:"
 
     def __init__(self, config, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.config = config
 
     def process(self, context, data):
+        logprefix = LdapAttributeStore.logprefix
+
+        # Initialize the configuration to use as the default configuration
+        # that is passed during initialization.
+        config = self.config
+        configClean = copy.deepcopy(config)
+        if 'bind_password' in configClean:
+            configClean['bind_password'] = 'XXXXXXXX'    
+
+        satosa_logging(logger, logging.DEBUG, "{} Using default configuration {}".format(logprefix, configClean), context.state)
+
+        # Find the entityID for the SP that initiated the flow
+        try:
+            spEntityID = context.state.state_dict['SATOSA_BASE']['requester']
+        except KeyError as err:
+            satosa_logging(logger, logging.ERROR, "{} Unable to determine the entityID for the SP requester".format(logprefix), context.state)
+            return super().process(context, data)
+
+        satosa_logging(logger, logging.DEBUG, "{} entityID for the SP requester is {}".format(logprefix, spEntityID), context.state)
+
+        # Examine our configuration to determine if there is a per-SP configuration
+        if spEntityID in self.config:
+            config = self.config[spEntityID]
+            configClean = copy.deepcopy(config)
+            if 'bind_password' in configClean:
+                configClean['bind_password'] = 'XXXXXXXX'    
```

Comment:
No. The intent is to log the configuration at the DEBUG level but blank out any bind password included in the configuration so that passwords are never logged.